maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -88,4 +95,11 @@
       </plugin>
     </plugins>
   </build>
+  
+  <properties>
+  	<!-- jgit version: 6.0.0.202111291000-r   need java 11
+         jgit version: 5.13.0.202109080827-r  need java  8  
+  	-->
+  	<org.eclipse.jgit.version>5.13.0.202109080827-r</org.eclipse.jgit.version>
```

Comment:
Since this requires Java 8, it won't happen in 1.x. Can this be done with an older version?

Reply:
It is not neccesary. I just tried to upgrade some very old dependencies. Git is evolving, JGIT shurely fixes a lot of bugs. Maven also moved to Java 8. It is paradox to fix something and keep downward compatibility while everyone moving forward. 

Are there plans to upgrade the plugins to 3.x?

But my heart whont bleed if keeping the old dependency.

Reply:
The 1.x line will remain on Java 7. 2.x will move to Java 8. So please make a decision how imporant this is for you then I can continue to review your PR.